### PR TITLE
OpenMap(URL) remove faulty webmap from sample

### PR DESCRIPTION
**Pull request by Luiy0**
### Description
Issue [here](https://devtopia.esri.com/runtime/java/issues/7803#event-21158260)

- Remove the webmap for the [world-terrestrial-ecosystems](https://arcgisruntime.maps.arcgis.com/home/item.html?id=926a206393ec40a590d8caf29ae9a93e) as it's not displaying correctly
- Update the sample image and metadata
- `v200.develop` branch

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by Luiy0** [comment]
Hi @jenmerritt, can you do a first review please?

File: map/open-map-url/README.metadata.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
         "PortalItem"
     ],
     "snippets": [
-        "src/main/java/com/esri/samples/open_map_url/OpenMapURLSample.java"
+      "src/main/java/com/esri/samples/open_map_url/OpenMapURLSample.java",
+      "src/main/java/module-info.java"
```
Comment (jenmerritt) [inline map/open-map-url/README.metadata.json]:
```suggestion
      "src/main/java/com/esri/samples/open_map_url/OpenMapURLSample.java"
```
I don't think we've been putting module-info in here?

**Review by jenmerritt — COMMENTED** [review]
Looks good just one thing to check before I approve, thanks!

File: map/open-map-url/README.metadata.json (hunk 1 above)
Comment (Luiy0) [reply map/open-map-url/README.metadata.json]: Removed. Will look at updating the script to exclude this from the metadata style checks.

File: map/open-map-url/README.metadata.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
         "PortalItem"
     ],
     "snippets": [
-        "src/main/java/com/esri/samples/open_map_url/OpenMapURLSample.java"
+      "src/main/java/com/esri/samples/open_map_url/OpenMapURLSample.java"
```
Comment (jenmerritt) [inline map/open-map-url/README.metadata.json]: Correct the spacing to as it was before

**Review by jenmerritt — APPROVED** [review]
I'll approve so as to not block you - just reset the spacing there. You can proceed with merging this despite the check failure since we know why, unless you come up with / have time for a quick solution there.

**Comment by Luiy0** [comment]
Ready for a second look @Rachael-E, thanks.

**Review by Rachael-E — APPROVED** [review]
thanks @Luiy0 , these change look good

**Pull request merged**